### DISCUSSION TITLE
WARN slave/slave_startup process has 65536 max file descriptor limit

### DISCUSSION THREAD

**Discussion by Robdewild** [AzuraCast/AzuraCast#4685] [Q&A]
Using docker.
Azuracast icecastlog constantly shows following error:
WARN slave/slave_startup process has 65536 max file descriptor limit

It goes on forever….
Program seems to work without further issues, but this is not good, i think.
Any ideas?

**Comment by Vaalyn** [comment]
This log entry is normal.

In the Icecast-kh fork of Icecast that AzuraCast is using this message is written to the logs just to show what the current file descriptor limit is. This message has caused quite a lot of questions from users since the `WARN` is kinda confusing since it's not really a warning in itself but we can't just disable the warning as far as I know.

**Comment by Robdewild** [comment]
Thanks for the quick response.
I’m happy👍

via 
Mobiel
